app/src/main/java/com/vasilevkin/catsanddogs/models/localModels/Animal.kt

```diff
@@ -0,0 +1,6 @@
+package com.vasilevkin.catsanddogs.models.localModels
+
+data class Animal(
+    val title: String?,
```

Comment:
you can use default param like val title: String? = null

Reply:
Fixed.